### PR TITLE
Pass opensea key from env

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: node/status_node_services.go [hunk 1]
```diff
@@ -52,6 +52,7 @@ var (
 	// ErrRPCClientUnavailable is returned if an RPC client can't be retrieved.
 	// This is a normal situation when a node is stopped.
 	ErrRPCClientUnavailable = errors.New("JSON-RPC client is unavailable")
+	OpenseaKeyFromEnv string
```
Comment (cammellos) [inline node/status_node_services.go:58]: Maybe add a comment that the variable is injected at build time, so that's it's more explicit

**Review by cammellos — APPROVED** [review]

**Comment by cammellos** [comment]
@jakubgs we need to inject the variable at build time, so when compiling we need do something like:
```
go build -ldflags "-X node.OpenseaKeyFromEnv=$OPENSEA_KEY"
```

is that a change that needs to be made on your side or should be in this PR?

**Comment by rasom** [comment]
@cammellos I added it on status-react side, @churik tested it seems working. I requested @jakubgs's review there 

https://github.com/status-im/status-react/pull/13144

File: node/status_node_services.go (hunk 1 above)
Comment (rasom) [reply node/status_node_services.go:58]: ok

Comment (rasom) [reply node/status_node_services.go:58]: @cammellos done

File: node/status_node_services.go [hunk 2]
```diff
@@ -52,6 +52,8 @@ var (
 	// ErrRPCClientUnavailable is returned if an RPC client can't be retrieved.
 	// This is a normal situation when a node is stopped.
 	ErrRPCClientUnavailable = errors.New("JSON-RPC client is unavailable")
+	// Opensea Key passed from on env var during build time
```
Comment (cammellos) [inline node/status_node_services.go]: Linting will complain, it needs to be the same name

**Pull request merged**